oat-sa/extension-tao-delivery
# fix: TR-3034. Change icons for RTL mode

Related to: https://oat-sa.atlassian.net/browse/TR-3034

How to test:
1. Add test taker with Arabic language
2. Publish test and assign created user
3. Check the icons styles according to ACs
